src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Books = () => (
```

Comment:
- [ ] You created your component in the `src/redux/books` folder. The **redux** directory is for redux-related configurations like actions creators, action types, store and reducers hence, at this point this particular file should be empty. Kindly extract this **`Books`** component to the `src/components` folder.